python/ray/rllib/env/vector_env.py

```diff
@@ -101,3 +109,38 @@ def vector_step(self, actions):
     @override(VectorEnv)
     def get_unwrapped(self):
         return self.envs
+
+
+class _RemoteVectorizedGymEnv(_VectorizedGymEnv):
```

Comment:
Consider extending VectorEnv directly, since you don't seem to use much of the functionality of VectorizedGymEnv

Reply:
Well, I do reuse constructor and get_unwrapped, no need to copy those. I would leave it like this.